exercises/J-functions/exercise2.js

```diff
@@ -1,5 +1,6 @@
 function triple(number) {
   // complete function here
+  return 3 * number;
```

Comment:
Awesome, this works also.

I would again advise using parenthesis to make it more clear, also it is very inconsequential but often we write it as `(number * 3)` instead of `3 * number` but it really doesn't matter. What you did is great 👍